### PR TITLE
fasthenry-wr 3.0 (new formula)

### PR DESCRIPTION
**Pull request by rob-deutsch**
First commit of fasthenry-wr software

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fasthenry-wr.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class FasthenryWr < Formula
+  desc "3D inductance computation program which handles superconducting conductors"
+  homepage "http://www.wrcad.com/freestuff.html"
+  url "http://www.wrcad.com/ftp/pub/fasthenry-3.0wr-071720.tar.gz"
+  version "3.0-071720"
```
Comment (carlocab) [inline Formula/fasthenry-wr.rb:5]: How does one know the correct version based on the tarball `url`? Do you always just remove the letters in between the digits?

File: Formula/fasthenry-wr.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  sha256 "161943c7d63ac3cf7b615b048a33ed77a0d12164fde461859e34b61ca4e63a73"
+  license "MIT"
+
+  def install
+    system "make", "all"
+    bin.install "bin/fasthenry"
```
Comment (carlocab) [inline Formula/fasthenry-wr.rb:11]: Is there no `make install`?

File: Formula/fasthenry-wr.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      *+ contact line (-2,-2,0,5,2,0,1,1)
+      *+ contact point (1,1,0,0.1,0.1)
+      + n_in (0.0,0,0)
+      + n_out (1.0,1,0)
+
+      .external n_in n_out
+      .freq fmin=0 fmax=1e9 ndec=1
+      .end
+    EOS
+    system "#{bin}/fasthenry", "test.inp"
```
Comment (carlocab) [inline Formula/fasthenry-wr.rb:32]:
```suggestion
    system bin/"fasthenry", "test.inp"
```
Would be good to maybe check some output here, if possible.

File: Formula/fasthenry-wr.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+class FasthenryWr < Formula
```
Comment (carlocab) [inline Formula/fasthenry-wr.rb:1]: Is this the same software as https://repology.org/project/fasthenry/versions?

File: Formula/fasthenry-wr.rb (hunk 2 above)
Comment (rob-deutsch) [reply Formula/fasthenry-wr.rb:11]: Correct, there is no `make install`.

File: Formula/fasthenry-wr.rb (hunk 1 above)
Comment (rob-deutsch) [reply Formula/fasthenry-wr.rb:5]: "wr" refers to the fact that this version of fasthenry is a fork from Whiteley Research Inc.

File: Formula/fasthenry-wr.rb (hunk 4 above)
Comment (rob-deutsch) [reply Formula/fasthenry-wr.rb:1]: Yes and no. They are all `fasthenry`, but even on that repology site it looks like they all might be different forks of `fasthenry`.

File: Formula/fasthenry-wr.rb (hunk 1 above)
Comment (carlocab) [reply Formula/fasthenry-wr.rb:5]: Ok, but that answers neither of my questions.

Comment (rob-deutsch) [reply Formula/fasthenry-wr.rb:5]: Apologies for my oversight.

The tarball has a version number and a date, and both of these make up the version number. It looks like version number is always 3.0, but the date changes. 

(An argument could be made that this package should be called `fasthenry3`, but I haven't seen any evidence that anyone else uses this number).

The letters in between the digits can and should be removed. They just indicate that its the wrcard fork of `fasthenry`, which seems to be the most popular fork.

File: Formula/fasthenry-wr.rb (hunk 3 above)
Comment (rob-deutsch) [reply Formula/fasthenry-wr.rb]: Agreed, in theory, but I don't understand `fasthenry` well enough to check the output. I need to install it on macOS for a totally different app that knows how to read the output from `fasthenry`.

Is that okay?

**Pull request closed without merging**